### PR TITLE
Feature - add 'JobId' to log messages in the Azure Service Bus message pump

### PR DESCRIPTION
**Pull request by stijnmoreels**
Closes https://github.com/arcus-azure/arcus.messaging/issues/58

### CODE REVIEW

**Comment by tomkerkhove** [comment]
We should have a look if .NET Core allows us to define it once and add it to all following messages.

**Comment by stijnmoreels** [comment]
> We should have a look if .NET Core allows us to define it once and add it to all following messages.

We could use instead an `EventId` for this, but then the `JobId` should be an integer.

**Comment by tomkerkhove** [comment]
Doesn't sound ideal imo

**Comment by stijnmoreels** [comment]
> Doesn't sound ideal imo

So... the current PR is good then for you?

**Comment by tomkerkhove** [comment]
Not really, it should automatically add jobid for all messages AFAIK you can do that with ILogger

File: src/Arcus.Messaging.Pumps.ServiceBus/AzureServiceBusMessagePump.cs [hunk 1]
```diff
@@ -163,7 +192,7 @@ protected virtual async Task CompleteMessageAsync(string lockToken)
 
             if (_messageReceiver == null)
             {
-                throw new InvalidOperationException("Message receiver is not initialized yet.");
+                throw new InvalidOperationException($"[Job: {JobId}] Message receiver is not initialized yet");
```
Comment (tomkerkhove) [inline src/Arcus.Messaging.Pumps.ServiceBus/AzureServiceBusMessagePump.cs]: We shouldn't add these prefixes to our exceptions

Comment (stijnmoreels) [reply src/Arcus.Messaging.Pumps.ServiceBus/AzureServiceBusMessagePump.cs]: 👍

**Pull request merged**